service/src/main/java/com/ge/predix/acs/config/PolicyEvaluationCacheConfig.java

```diff
@@ -17,22 +23,27 @@
     @Value("${ENABLE_CACHING:false}")
     private boolean cachingEnabled;
 
-    @Value("${ENABLE_CIRCUIT_BREAKER:false}")
-    private boolean hystrixCircuitBreakerEnabled;
+    @Autowired
+    private Environment environment;
 
     @Bean
-    public PolicyEvaluationCacheCircuitBreaker cache() {
+    public PolicyEvaluationCache cache() {
         if (!this.cachingEnabled) {
-            LOGGER.info("Caching is disabled.");
-            return new BasicPolicyEvaluationCacheCircuitBreaker();
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Caching disabled");
```

Comment:
Should we log as info that cache is disable?
LOGGER.info("Policy evaluation caching is disabled.");